uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/leave-request-api.js

```diff
@@ -278,7 +277,7 @@ define([
           leave_request_id: leaveRequestID,
           text: comment.text,
           contact_id: comment.contact_id,
-          created_at: moment(comment.created_at, sharedSettings.serverDateFormat).format(sharedSettings.serverDateTimeFormat)
+          created_at: comment.created_at
```

Comment:
> We would create the time in proper format when the comment is generated from UI, so changed the logic . Thanks for pointing it out.

It will be done when user comments.